engine-tests/src/test/java/org/terasology/documentation/apiScraper/ApiComparator.java

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright 2017 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.documentation.apiScraper;
+
+import com.google.common.collect.Multimap;
+import com.google.common.collect.Multimaps;
+import org.terasology.documentation.apiScraper.util.ApiMethod;
+
+import java.io.*;
```

Comment:
Avoid star imports - make sure you set up Checkstyle for IDE highlight of errors, this file lights up like a 🎄 :-)